twc/commands/domain.py

```diff
@@ -0,0 +1,539 @@
+    fmt.printer(
+        response,
+        output_format=output_format,
+        func=(lambda x: print("Complite!") if x.status_code == 204 else None),
+    )
+
+
+# ------------------------------------------------------------- #
+# $ twc domain record list domain.io                            #
+# ------------------------------------------------------------- #
+
+
+def print_domain_record_list(
+    response: Response,
+    not_ignore_subdomains,
+    response_domain,
+    filters: Optional[str] = None,
+):
+    """Print domain records."""
+    domains = response.json()["dns_records"]
+
+    if not not_ignore_subdomains:
+        domains = filter(lambda x: "subdomain" not in x["data"], domains)
+
+    if filters:
+        domains = fmt.filter_list(domains, filters)
+
+    table = fmt.Table()
+    table.header(
+        [
+            *(["SUB"] if not_ignore_subdomains else []),
+            "TYPE",
+            "VALUE",
+            "ID",
+        ]
+    )
+
+    for domain_json in domains:
+        table.row(
+            [
+                *(
+                    [domain_json["data"]["subdomain"]]
+                    if "subdomain" in domain_json["data"]
+                    else [""]
+                    if not_ignore_subdomains
+                    else []
+                ),
+                domain_json["type"],
+                domain_json["data"]["value"],
+                domain_json["id"],
+            ]
+        )
+    print(f"DOMAIN: {response_domain}\n")
+    table.print()
+
+
+@domain_record.command("list", "ls")
+def domain_records_list(
+    domain_name: str,
+    verbose: Optional[bool] = verbose_option,
+    config: Optional[Path] = config_option,
+    profile: Optional[str] = profile_option,
+    output_format: Optional[str] = output_format_option,
+    filters: Optional[str] = filter_option,
+    not_ignore_subdomains: bool = typer.Option(
+        False,
+        "--not-ignore-subdomains",
+        case_sensitive=False,
+        help="Not ignoring TXT subdomain records.",
+    ),
+):
+    """List dns records on domain."""
+    client = create_client(config, profile)
+    response = client.get_domain_dns_records(domain_name)
+
+    fmt.printer(
+        response,
+        output_format=output_format,
+        filters=filters,
+        func=print_domain_record_list,
+        not_ignore_subdomains=not_ignore_subdomains,
+        response_domain=domain_name,
+    )
+
+
+# ------------------------------------------------------------- #
+# $ twc domain record rm domain.io record_id                    #
+# ------------------------------------------------------------- #
+
+
+@domain_record.command("delete", "rm")
+def domain_remove_dns_record(
+    domain_name: str,
+    record_id: int,
+    verbose: Optional[bool] = verbose_option,
+    config: Optional[Path] = config_option,
+    profile: Optional[str] = profile_option,
+    output_format: Optional[str] = output_format_option,
+    filters: Optional[str] = filter_option,
+):
```

Comment:
Hi, @ranmasootome! Thanks for your PR, it really helped me in testing my TW resources before official release. You really saved a lot of my time! The only issue I have found - is on record deletion, the cli command fails with error:

```
ff6256c53468:~# twc domain record rm mydomain.com 53749793
Traceback (most recent call last):

  File "/usr/local/bin/twc", line 8, in <module>
    sys.exit(cli())
             ^^^^^

  File "/usr/local/lib/python3.11/site-packages/twc/commands/domain.py", line 340, in domain_remove_dns_record
    fmt.printer(

  File "/usr/local/lib/python3.11/site-packages/twc/fmt.py", line 131, in printer
    to_print.print(output_format, func=func, **kwargs)

  File "/usr/local/lib/python3.11/site-packages/twc/fmt.py", line 116, in print
    func(self._data, **kwargs)

TypeError: domain_remove_dns_record.<locals>.<lambda>() got an unexpected keyword argument 'response_domain'
```
Regardless of error response, the record is being deleted.